ext/date/tests/DateTime_verify.phpt

```diff
@@ -31,7 +31,7 @@ object(ReflectionClass)#%d (1) {
   string(8) "DateTime"
 }
 ..and get names of all its methods
-array(19) {
+array(20) {
```

Comment:
This is a stupid test that we probably should remove. @nikic ?

Reply:
We should... done in https://github.com/php/php-src/commit/8e12f04b5e48645997be9da51bf16813cca5a402.